src/features/device-config/device-config.ts

```diff
@@ -120,6 +121,8 @@ export const generateConfig = async (
 				delta: `https://${DELTA_HOST}`,
 				registry: REGISTRY2_HOST,
 				vpn: VPN_HOST,
+				// If undefined, balena-device-config won't generate a logs endpoint
+				logs: LOGS_HOST ? `https://${LOGS_HOST}` : undefined,
```

Comment:
NIT: I guess it could even be v so that we don't set empty properties at all (even though express might be removing them during serialization anyway):
```suggestion
				...(LOGS_HOST != null && { logs: `https://${LOGS_HOST}` },
```

Reply:
`undefined` cannot be represented in JSON and so would be removed altogether